examples/PySDM_examples/Shipway_and_Hill_2012/simulation.py

```diff
@@ -34,12 +34,15 @@ def __init__(self, settings, backend=CPU):
             grid=(settings.nz,),
             size=(settings.z_max + settings.particle_reservoir_depth,),
         )
+
         self.env = Kinematic1D(
             dt=settings.dt,
             mesh=self.mesh,
             thd_of_z=settings.thd,
             rhod_of_z=settings.rhod,
             z0=-settings.particle_reservoir_depth,
+            z_part=None,
```

Comment:
these are the defaults, right? no need to add this change

Reply:
Yes, thanks for pointing this out!